### DISCUSSION TITLE
Adding external(uniSat) wallet to rainbowkit integration

### DISCUSSION THREAD

**Discussion by Lakszy** [rainbow-me/rainbowkit#2026] [General]
I am working on an app on which I can use Rainbow Kit integration 
I want to add uniSat wallet in that too
Does someone have any idea regarding this
Help would be much appreciated
Thanks!

**Comment by DanielSinclair** [comment]
Wagmi and RainbowKit currently only support the Ethereum ecosystem, but you could have a look at something like this library for BTC support: https://github.com/bob-collective/sats-wagmi
